linode/manager
# upcoming: [DI-20501] - Fix Demo feedbacks and missed changes across ACLP

## Description 📝
Fixed demo feedbacks and missed changes across ACLP

## Changes  🔄

1. Fixed Dashboard Select placeholder from **Select a Dashboard to Select Dashboard**
2. Fixed striking functionality interchangeably while clicking native legend as well as legend table row.
3. Enhanced Region Selection component to honor the placeholder prop being passed.
4. Pass fill option to line graph only if the chart type is area.
5. Pass queried resource option for choosing default selections as well on the CloudPulseCustomSelect component
6. Integrate manual refresh functionality
7. Fix color for close icon in the CloudPulseResourcesSelection component
8. Update labels in time duration from **Past to Last**

## Target release date 🗓️
02-09-2024

## Preview 📷
**Include a screenshot or screen recording of the change**

:bulb: Use `<video src="" />` tag when including recordings in table.

| Before  | After   |
| ------- | ------- |
|![Screenshot 2024-08-29 at 1 15 12 PM](https://github.com/user-attachments/assets/969e1b57-efb5-4276-9b4f-26aa44067908)|![Screenshot 2024-08-29 at 1 20 09 PM](https://github.com/user-attachments/assets/84f4bcc9-2cc2-413e-bcb7-ef6571e36bdb)|
|![Screenshot 2024-08-29 at 1 16 12 PM](https://github.com/user-attachments/assets/5b100d13-82cc-4003-b43d-5edefe7f0adb)|![Screenshot 2024-08-29 at 1 20 27 PM](https://github.com/user-attachments/assets/7d29e648-e871-4651-9428-af44599f4b59)|
|![Screenshot 2024-08-29 at 1 17 07 PM](https://github.com/user-attachments/assets/fa562bb4-e451-449c-9cef-c586e94e8f39)|![Screenshot 2024-08-29 at 1 21 09 PM](https://github.com/user-attachments/assets/bb54d463-28f1-42c7-8b31-f5b9976aca7a)|
|![Screenshot 2024-08-29 at 1 18 22 PM](https://github.com/user-attachments/assets/ec2454ba-3005-4dbc-9c17-4c72a73d006f)|![Screenshot 2024-08-29 at 1 20 48 PM](https://github.com/user-attachments/assets/2e0274bd-4d5c-4a3f-8a66-6102638b4e1a)|
|![Screenshot 2024-08-29 at 1 19 01 PM](https://github.com/user-attachments/assets/1b8e8029-dc33-46a5-8366-b3f7de524766)|![Screenshot 2024-08-29 at 1 21 31 PM](https://github.com/user-attachments/assets/460b58dc-90aa-464b-959a-bf47e1cd360d)|

## How to test 🧪

1. Enable mock service as some API endpoints are not available.
2. Click on the Monitor tab
3. You will be able to see the options and fixes
4. For global manual refresh, once you see the widgets rendered, click the refresh icon, you can see again the widgets are loaded.


## As an Author I have considered 🤔

*Check all that apply*
